theseus/classification/utilities/gradcam.py

```diff
@@ -51,12 +51,12 @@ def __init__(self,
                 if model_name.startswith(available_model):
                     model_name = available_model
                     break
-            else:
```

Comment:
This is not wrong, I think you get my idea wrong on this line of code. This is the structure of `for` ... `else` ... in python.

> In gradcam, maybe we should make a pass-through when we use any model from timm that's not in model_last_layers. For example, it won't plot the "validation gradcam" but it can still run the program normally, you know what i mean right? In current pipeline, it's gonna stop the program, which i think it's convenient to users who don't know how to correct it.

- These lines are meant to be used for this checkup, currently it only prints out error notification and breaks the program, if you want to run the program normally there must be some interactions between the `CAMWrapper` and the `VisualizerCallbacks`. I can't figure out a quick solution for this right now, do you have any idea.?